soh/soh/Enhancements/game-interactor/GameInteractor_Hooks.cpp

```diff
@@ -181,3 +181,9 @@ void GameInteractor_ExecuteOnUpdateFileNameSelection(int16_t charCode) {
 void GameInteractor_ExecuteOnSetGameLanguage() {
     GameInteractor::Instance->ExecuteHooks<GameInteractor::OnSetGameLanguage>();
 }
+
+// MARK: - System
+
+void GameInteractor_RegisterOnAssetAltChange(void (*fn)(void)) {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnAssetAltChange>(fn);
+}
```

Comment:
This is the first example of making a C bridge method for registering to a hook on the C side. Sticking it here to go with the other bridge methods.